Makefile

```diff
@@ -26,9 +31,24 @@ check-license:
 
 tools: $(TOOLS)
 
+man: $(MAN)
+
+all: $(TOOLS) $(MAN)
+
 $(TOOLS): oci-%:
 	go build -ldflags "-X main.gitCommit=${COMMIT}" ./cmd/$@
 
+.SECONDEXPANSION:
+$(MAN): %.1: cmd/$$*/$$*.1.md
+	go-md2man -in "$<" -out "$@"
```

Comment:
should we install `go-md2man` in `install.tools` https://github.com/opencontainers/image-tools/blob/master/Makefile#L60?